koku/masu/api/upgrade_trino/util/constants.py

```diff
@@ -0,0 +1,57 @@
+from enum import Enum
+
+# Update this to trigger the converter to run again
+# even if marked as successful
+CONVERTER_VERSION = "1"
+
+
+class ReprEnum(Enum):
+    """
+    Only changes the repr(), leaving str() and format() to the mixed-in type.
+    """
+
+
+# StrEnum is available in python 3.11, vendored over from
+# https://github.com/python/cpython/blob/c31be58da8577ef140e83d4e46502c7bb1eb9abf/Lib/enum.py#L1321-L1345
+class StrEnum(str, ReprEnum):
```

Comment:
We can ignore coverage on this since it's vendored code.
```suggestion
class StrEnum(str, ReprEnum):  # pragma: no cover
```